package.json

```diff
@@ -20,6 +20,7 @@
     "eslint": "eslint . --report-unused-disable-directives",
     "eslint-fix": "eslint --fix . --report-unused-disable-directives",
     "generate": "run-s generate:*",
+    "generate:distro-json-generate": "node tools/distro-json-generate.mjs",
```

Comment:
please revert this

Reply:
sorry for that again, must have forgotten to extract it with the refactor as requested